app/models/order_item.rb

```diff
@@ -2,5 +2,6 @@ class OrderItem < ApplicationRecord
   belongs_to :product
   belongs_to :order
 
-  validates :quantity, presence: true
+  #maybe add a unique combindation requirement of order id and product id
+  validates :quantity, presence: true, numericality: {only_integer: true, greater_than: 0}
```

Comment:
nice! this tells us that the quantity of items that can be purchased must be a number greater than 0 im guessing/